### PR TITLE
Market > Listings, Measure > Metrics

### PR DESCRIPTION
**Pull request by Lukewh**
# Done

- Renam the `market` endpoint `listing`
- Rename `market.html` to `listing.html`
- Rename the `measure` endpoint `metrics`
- Rename `measure.html` to `metrics.html`
- Update python function names
- Update tests
- Update references in templates

# Not done

- Update JS variables and function names

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account and click around:
- There should be no reference to 'market' or 'measure'
- Visit http://0.0.0.0:8004/account/snaps/<snap_name>/market and http://0.0.0.0:8004/account/snaps/<snap_name>/measure, you should be redirected to the appropriate page

### CODE REVIEW

File: tests/test_get_metrics.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 class MeasurePageNotAuth(BaseTestCases.EndpointGetLoggedOut):
```
Comment (tbille) [inline tests/test_get_metrics.py]: Update class name

Comment (Lukewh) [reply tests/test_get_metrics.py]: Done!

File: app.py [hunk 2]
```diff
@@ -345,7 +345,25 @@ def snap_release(snap_name):
...
-@app.route('/account/snaps/<snap_name>/market', methods=['POST'])
+@app.route('/account/snaps/<snap_name>/listing', methods=['POST'])
+@login_required
+def post_listing_snap(snap_name):
+    return publisher_views.post_listing_snap(snap_name)
+
+
+@app.route('/account/snaps/<snap_name>/market')
+@login_required
+def get_market_snap(snap_name):
```
Comment (tbille) [inline app.py]: let's make it more readable: 
```python
def get_market_snap(snap_name):
    return flask.redirect(
        "/account/snaps/{snap_name}/listing".format(
            snap_name=snap_name))
```

**Comment by tbille** [comment]
@Lukewh a little bit annoying but there is still a lot of `market` vars in the JS. (file market.js, in the html ...) Do you wanna update this now? Because I have the feeling that it will never be updated if we don't do it now...

**Comment by Lukewh** [comment]
@tbille I decided not to do it now (as mentioned in this PR description) because there are a lot of variables and functions that are 'marketX' which feels fragile to change when rushing for a release, especially when we're hopefully refactoring the listings page js in the coming month or two.

File: app.py [hunk 3]
```diff
@@ -345,7 +345,23 @@ def snap_release(snap_name):
...
+    return flask.redirect(
+        "/account/snaps/{snap_name}/listing".format(
+            snap_name=snap_name))
+
+
+@app.route('/account/snaps/<snap_name>/measure')
 @login_required
-def post_market_snap(snap_name):
-    return publisher_views.post_market_snap(snap_name)
+def get_measure_snap(snap_name):
```
Comment (bartaz) [inline app.py]: Shouldn't it be `get_metrics_snap` `get_listing_snap` - or maybe even `get_snap_metrics` `get _snap_listing`?

Comment (Lukewh) [reply app.py]: that's just the redirect to metrics, figured would leave them as the legacy name and keep the name formats the same throughout

**Pull request merged**